server/vulnerabilities/cpe_translations.json

```diff
@@ -0,0 +1,33 @@
+[
+  {
+    "match": {
+      "bundle_identifier": ["us.zoom.xos"],
+      "source": ["apps"]
+    },
+    "translation": {
+      "product": ["zoom", "meetings"],
+      "vendor": ["zoom"],
+      "target_sw": ["macos", "mac_os"]
+    }
+  },
+  {
+    "match": {
+      "name": ["/^ruby(@.*)?$/"],
+      "source": ["homebrew_packages"]
+    },
+    "translation": {
+      "product": ["ruby"],
+      "vendor": ["ruby-lang"]
+    }
+  },
+  {
+    "match": {
+      "name": ["/^node(@.*)?$/"],
+      "source": ["homebrew_packages"]
+    },
+    "translation": {
+      "product": ["node.js"],
+      "vendor": ["nodejs"]
+    }
+  }
+]
```

Comment:
Q: There's a clear log output in fleet that would allow us to identify what's not matching and then edit this file?
Maybe we could add docs for such a process (on a later PR).

Reply:
I'm not sure what you mean. Most software wouldn't match the translations, so it would be a lot of spam in the logs. Perhaps we could log at debug level if something matches instead.

Reply:
Sorry, meant to log the software that would not match the original CPE translations (not this one).

Reply:
Nvm, just read this: `translations are used to aid in cpe matching.` (I was thinking this in the wrong way.)